<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RgbWand</class>
 <widget class="QDialog" name="RgbWand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(245,245,245);</string>
   </property>
   <property name="text">
    <string>RGB Fenster</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="bilder.qrc">
     <normaloff>:/bilder/Icons/arrow_left_icon.png</normaloff>:/bilder/Icons/arrow_left_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius:10px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">background-color:rgba(255,255,255,100);
border:none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_toggle">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
background-color:rgba(0,0,0,0);
color:rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>An</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">background-color:rgba(255,255,255,100);
border:none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_einst">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;
background-color:rgba(0,0,0,0);
color:rgb(0,0,0);</string>
        </property>
        <property name="text">
         <string>Farbe
einstellen</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bilder/Icons/back_blue.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="bilder.qrc"/>
 </resources>
 <connections/>
</ui>
